.github/workflows/ci.yml

```diff
@@ -18,18 +18,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Setup JDK
-        uses: actions/setup-java@v3
-        with:
-          distribution: corretto
-          java-version: 11
```

Comment:
As [.tool-versions](https://github.com/guardian/fezziwig/blob/main/.tool-versions) currently specifies Java 21, removing the java version here means updating the version used by the app. If we're doing that I think it would be good to test with consumers of this library (e.g. content-api-models) to make sure they can still use the library when compiled with Java 21.

I think for this PR it'd be better to stick to Java 11 (by keeping setup-java and adding setup-sbt, I think?).

Reply:
This is just the version used for CI though, isn't it?  The [release workflow](https://github.com/guardian/fezziwig/blob/main/.github/workflows/release.yml) uses the [Java version stored in .tool-versions](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/configuration.md#java-version).

Reply:
Oh thank you, that’s a good point! In that case we might as well merge this, since we’re already using Java 21. It would be good to see (in a subsequent PR) whether we should add a `--release` option to `scalacOptions`, to support consumers using Java 11.